### PR TITLE
Introduces rounding settings support, modifies how the default rounding displays

### LINKED ISSUE #929: Introduce configurable rounding for amounts displayed on the UI
## Abstract

Currently, all the amounts displayed in the UI use rounding. More specifically it uses [`BigNumber.ROUND_HALF_UP`](https://mikemcl.github.io/bignumber.js/#round-half-up) which is BigNumber.js's default mode. 

It would be desirable to provide an option so the users can select the preferred rounding mode.

### LINKED ISSUE #2479: Rounding error in balance amounts
Amounts are _nearly_ always incorrectly rounded up, such that a balance of 0.011 is rounded up to 0.02, when the floating point precision is set to 2. I have added a particular blockchain balance which correctly shows 0.011 when I have a floating point precision of 3, but when I change it to 2, the balance is shown as 0.02. 

The incorrect rounding is shown here:
- If you go to Accounts & Balances, and click on the specific asset. Under 'amount' in the table it shows the wrong amount.
- The total owned on that same asset page is also wrong, apparently based on the rounded numbers in the table below.
- On the Dashboard under assets it also has the wrong total.

However, the _correctly_ rounded total also makes an appearance:
- If you go to Accounts & Balances and look at the table of 'blockchain balances per asset' (so without clicking on any particular asset), there you see a correctly rounded total.

Operating system: Windows 10
Rotki version: 1.14.2

### PR DESCRIPTION
**Pull request by kelsos**
Closes #929
Closes #2479

### CODE REVIEW

**Comment by kelsos** [comment]
Note to self -> Changelog entries are pending (I need to add them after we discuss and finalize the changes in this PR)

**Review by LefterisJP — COMMENTED** [review]
The difference in rounding method is in cases where it's close to zero, to not show zero, but 0.01 (or whatever the rounding decimals is), right?

Such as here (very small number super close to zero).




Seems to work. Not sure on the `<` symbol before the number. But I can't think of a better alternative. The other one I think `~` does not indicate less than.

Perhaps users or a designer can come back with feedback and we can adjust the symbol then.

As you also put a reminder to yourself you would need to also add changelog :smile:

**Comment by kelsos** [comment]
@LefterisJP done. I added two entries since there are two distinct issues in the PR

File: docs/changelog.rst [hunk 1]
```diff
@@ -12,6 +13,7 @@ Changelog
 * :feature:`2513` Users can now add/edit/delete all types of assets, not only ethereum tokens.
 * :feature:`2424` Users will now see a progress bar while the automatic update is downloading, and proper notification messages in case of failure.
 * :feature:`2515` Users will now be able to navigate back from the assets page using a button.
+* :bug:`2479` Users will now see a < (less than) symbol in front off any amount with trailing decimals when rounding upwards is used.
```
Comment (LefterisJP) [inline docs/changelog.rst:16]:
```suggestion
* :bug:`2479` Users will now see a < (less than) symbol in front of any amount with trailing decimals when rounding upwards is used.
```

**Review by LefterisJP — COMMENTED** [review]
Checked it out. Works. Looks okay, but the location of the settings for rounding can be improved.

We already have a section related to amounts here:




I would say place it in that same section as it logically also belongs there.

**Comment by kelsos** [comment]
@LefterisJP done

**Review by LefterisJP — APPROVED** [review]
LGTM kelsos. Thanks!

**Pull request merged**